openapi/v1/schemas/job/ImportAFileRequest.yaml

```diff
@@ -13,7 +14,11 @@ properties:
   dataLink:
     example: https://mybucket.s3.amazonaws.com/my-experiment/my-object.pdf
     type: string
-    description: Provide a link to a file to import it as an attachment. The file will be associated with a study.
+    description: Provide a link to a file to import it as an attachment. The file will be associated with a study and searchable by its name and metadata.
+  metadataLink:
+    example: https://mybucket.s3.amazonaws.com/my-experiment/my-object-metadata.tsv
+    type: string
+    description: It is also possible to optionally supply the URL of a metadata file. This metadata will be used as the original metadata for the created objects. The file is expected to contain a single record with metadata describing the uploaded attachment.
```

Comment:
>The file is expected to contain a single record with metadata describing the uploaded attachment.
To be honest I don't understand what does it mean

Reply:
parameters description from Alexey:

metadataLink: Specifies the URL of a metadata file to be used as the original metadata for the created objects. The file must contain a single record describing the uploaded attachment.

dataLink: Specifies a link to a file to import as an attachment. The file will be associated with a study and will be searchable by its name and metadata.

@m-smazhevsky, @egor-alexandrov do you have any objections?